styles/style.css

```diff
@@ -0,0 +1,80 @@
+h1 {
+  font-family: 'Noto Serif JP', sans-serif;
```

Comment:
maybe redundant to upload backup font bc you already imported the font.. but it's ok